q2_feature_table/_filter.py

```diff
@@ -29,8 +30,8 @@ def f(data_vector, id_, metadata):
 _other_axis_map = {'sample': 'observation', 'observation': 'sample'}
 
 
-def _filter(table, min_frequency, max_frequency, min_nonzero, max_nonzero,
-            metadata, where, axis, exclude_ids=False):
+def _filter_table(table, min_frequency, max_frequency, min_nonzero,
```

Comment:
I renamed this util to disambiguate from the other types of filtering this module now performs.